sbd.spec

```diff
@@ -97,6 +105,17 @@ rm -rf %{buildroot}
 %postun
 %systemd_postun sbd.service
 %systemd_postun sbd_remote.service
+%else
+/sbin/chkconfig --add sbd_helper || :
+
+%preun
+/sbin/service sbd_helper stop  &>/dev/null || :
+/sbin/service sbd_remote_helper stop  &>/dev/null || :
+if [ $1 -eq 0 ]; then
+    # Package removal, not upgrade
+    /sbin/chkconfig --del sbd_helper || :
+    /sbin/chkconfig --del sbd_remote_helper || :
+fi
```

Comment:
both have their pros & cons I guess ...
the %postun present just in the systemd-case I had though it would look
somehow asymmetric.
Thus I had gone for the same as in pacemaker ...